ast/ddl.go

```diff
@@ -192,18 +192,27 @@ func (n *DropDatabaseStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
-// IndexColName is used for parsing index column name from SQL.
-type IndexColName struct {
+// KeyPartSpecifications is used for parsing index column name or index expression from SQL.
+type KeyPartSpecification struct {
```

Comment:
I think it's better to use the name with the "Index" prefix because our previous index-related structures were prefixed with "Index" instead of "Key".